RELEASE.txt

```diff
@@ -5,10 +5,14 @@ This is the procedure for generating a new Dafny release.
 
 1) In an up-to-date repository, with the master branch checked out
    and up-to-date (e.g., git pull upstream)
+   checkout a new branch from master (e.g., git checkout -b branchname master)
    cd to the repo root (i.e., dafny, which contains Source, Binaries, etc.)
 
 2) Select a version number (descriptor) $VER (e.g., "3.0.0" or "3.0.0-alpha")
    and select an internal version number (e.g., "3.0.0.30203")
+   The last section is in format "ymmdd" format
+   where y is years since 2018 and then release date in "mmdd"
+   (e.g., Release on January 12th 2022  2018 + 4 years and date in "mmdd" - x.y.z.40112)
```

Comment:
```suggestion
   and select an internal version number (e.g., "3.0.0.30203").
   The last section is in "ymmdd" format, where "y" is the number of years
   since 2018 and "mmdd" the month and day portions of the release date (e.g.,
   release on January 12th, 2022 -> x.y.z.40112).
```